Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -835,6 +841,8 @@ - (void)applicationDidBecomeActive {
             }
         });
     }
+    
+    [self bnc_addDebugGestureRecognizer];
```

Comment:
You'll need a corresponding remove in the `applicationWillResignActive` to avoid multiple gestureRecognizers being added.